birdhouse/optional-components/prometheus-longterm-metrics/prometheus.yml.template

```diff
@@ -0,0 +1,18 @@
+global:
+  external_labels:
+    instance_name: prometheus-longterm-metrics
+
+scrape_configs:
+  - job_name: 'federate'
+    scrape_interval: ${PROMETHEUS_LONGTERM_SCRAPE_INTERVAL}
+
+    honor_labels: true
+    metrics_path: '/prometheus/federate'
+
+    params:
+      'match[]':
+        - '{group="longterm-metrics"}'
+
+    static_configs:
+      - targets:
+        - 'prometheus:9090'
```

Comment:
If you make this target list a template expansion variable, then we can easily add more hosts and override this default local Prometheus and points to other hosts, ex:
```
- existing_pavics_1:9090
- existing_pavics_2:9090
- ...
```

This assume I open up the Prometheus port on `existing_pavics_1` and `existing_pavics_2` and I handle manually the inclusion of `prometheus.rules` on on `existing_pavics_1` and `existing_pavics_2`.

Then you can remove the dependency on `./components/monitoring`.  

Just add a note in the README to enable `./components/monitoring` together with this `./optional-components/prometheus-longterm-metrics` component.

Reply:
I think this is for a separate discussion and out of the scope of this PR.

We can discuss the best way to handle multiple targets later on but at the very least we need to consider the security implications of recommending users open up additional ports.

I really think that any code that handles monitoring multiple prometheus endpoints should go in a separate repo.